apps/frontend/src/components/Sidebar.tsx

```diff
@@ -97,33 +59,21 @@ const SidebarLink = ({
 const Sidebar: React.FC = () => {
   const pathname = usePathname()
 
-  const router = useRouter()
-
-  const { session } = useAuth()
-
   const { showModal } = useModal()
 
-  const [lastSpaceRoute, setLastSpaceRoute] = useState<string | null>(null)
-  const [isHovered, setIsHovered] = useState(false) // State for hover
-  const mouseY = useMotionValue(Infinity)
+  const [isExpanded, setIsExpanded] = useState(false)
+  const [holdIsExpanded, setHoldIsExpanded] = useState(false)
```

Comment:
I don't think this state is required here. You can use isExpanded only

Reply:
you are right haha thank you